### PR TITLE
environment id support added

### PR DESCRIPTION
**Pull request by in4s**

### CODE REVIEW

**Comment by sklmx** [comment]
@in4s update utils to [v1.0.1](https://github.com/qase-tms/qase-php-client-utils/releases/tag/v1.0.1), please

File: src/Reporter.php [hunk 1]
```diff
@@ -73,6 +74,9 @@ public function executeBeforeFirstTest(): void
         );
 
         $this->validateProjectCode();
+        if ($this->config->getEnvironmentId() !== null) {
```
Comment (sklmx) [inline src/Reporter.php:77]: to make the method clearer move this condition to **validateEnvironmentId**

**Review by sklmx — CHANGES_REQUESTED** [review]

File: src/Reporter.php (hunk 1 above)
Comment (in4s) [reply src/Reporter.php]: done

**Comment by in4s** [comment]
> @in4s update utils to [v1.0.1](https://github.com/qase-tms/qase-php-client-utils/releases/tag/v1.0.1), please

Do you mean updating this library locally on my PC or elsewhere?

File: src/Reporter.php (hunk 1 above)
Comment (sklmx) [reply src/Reporter.php]: all good, but use "early return"

**Comment by sklmx** [comment]
> > @in4s update utils to [v1.0.1](https://github.com/qase-tms/qase-php-client-utils/releases/tag/v1.0.1), please
> 
> Do you mean updating this library locally on my PC or elsewhere? @sklmx if you mean changing this line: `"qase/php-client-utils": "^1.0"` I thought it is not necessary since it should use the latest patch version anyway.

This only works for a fresh install, you can still only update **qase-phpunit** which uses features from version 1.0.1. Also, the version in composer.lock is still fixed to 1.0.0.

File: src/Reporter.php (hunk 1 above)
Comment (in4s) [reply src/Reporter.php]: done

**Review by sklmx — APPROVED** [review]

**Pull request merged**